### PR TITLE
Fixing Vte error

### PR DESCRIPTION
**Pull request by Aniket21mathur**
Fixing TypeError: Vte.Terminal.feed_child() takes exactly 2 arguments (3given)
.The first argument is self, second is command and third is length of
command so removing len(command).

Fixes https://github.com/sugarlabs/riverhex-activity/issues/6

### CODE REVIEW

File: activity.py [hunk 1]
```diff
@@ -70,9 +70,9 @@ def __init__(self, handle):
         vt.show()
 
         if platform.machine().startswith('arm'):
-            vt.feed_child("echo \"RiverHex, this activity won't run on this computer\"; sleep 10; exit\n", -1)
+            vt.feed_child("echo \"RiverHex, this activity won't run on this computer\"; sleep 10; exit\n")
         else:
-            vt.feed_child("cd $SUGAR_BUNDLE_PATH/bin; ./bin/riverhexactivity-activity; exit\n", -1)
+            vt.feed_child("cd $SUGAR_BUNDLE_PATH/bin; ./bin/riverhexactivity-activity; exit\n")
```
Comment (Hrishi1999) [inline activity.py]: Remove the `exit` command. It force quits the actvity. Also, change `./bin/riverhexactivity-activity` to `./riverhexactivity-activity` (args required)

**Comment by Hrishi1999** [comment]
See (https://github.com/sugarlabs/riverhex-activity/pull/8/commits/3511bd31f177d7eb06d443e2c4e56cc7f3f72ae9#r251224306)

**Comment by Aniket21mathur** [comment]
Thanks @Hrishi1999. Made the above changes :smile: .

**Comment by quozl** [comment]
Fix https://github.com/sugarlabs/riverhex-activity/issues/5 first, thanks.  Closing.

**Pull request closed without merging**